packages/hooks/src/useAvailableWallets.ts

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import { useFetch } from '@deriv/api';
-import { useStore } from '@deriv/stores';
-import useWalletsList from './useWalletsList';
 import useAuthorize from './useAuthorize';
+import useWalletAccountsList from './useWalletAccountsList';
+import useCurrencyConfig from './useCurrencyConfig';
 
 const useAvailableWallets = () => {
-    const { client, ui } = useStore();
-    const { is_dark_mode_on } = ui;
-    const { is_crypto } = client;
-    const { data } = useAuthorize();
+    const { data, isSuccess, isLoading } = useAuthorize();
 
     // @ts-expect-error Need to update @deriv/api-types to fix the TS error
     const { data: account_type_data, ...rest } = useFetch('get_account_types', {
         payload: { company: data?.landing_company_name === 'virtual' ? 'svg' : data?.landing_company_name },
-        options: { enabled: Boolean(data?.landing_company_name) },
+        options: { enabled: Boolean(data?.landing_company_name) && isSuccess && !isLoading },
```

Comment:
@sergei-deriv why we need to check the `isSuccess && !isLoading`? the `get_account_types` is already in the `AUTH_REQUIRED_ENDPOINTS` list and will wait for the authorize

Reply:
@mahdiyeh-deriv it's because we have this code:

<img width="1074" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/4cbc1d42-8fac-4fa2-80f0-9bd810695591">

Cause we already have `enabled` property in this case we will not check `isAuthorized` if `Boolean(data?.landing_company_name)` is true. That's why I need to check authorize again.

Reply:
Changed `??` to `&&` in `useFetch` and delete check for `isSuccess && !isLoading`